### PR TITLE
#97  Created a single form in useredit page to update or delete user

### PR DESCRIPTION
**Pull request by shriaas2898**
Fixes a part of #97
* The user edit page displays a list of user (currently displayed using table, will be updated in follow up) and option to edit user which redirects to a form to edit user details. 
* Removed table with forms for editing user.
* Created a single form to user information.
* Added option to delete user.
* Removed echo statements inside the function definition from `books.php` and `user.php`.
* Changed php redirects to js redirects.

@rahulsavaria @guptahemant @anjali-rathod please review.

Made the changes requested in #128  here

### CODE REVIEW

File: public-html/asset/scss/components/form.scss [hunk 1]
```diff
@@ -108,12 +95,9 @@ button {
...
 
 /* Darker background on mouse-over */
-.iconbtnedit:hover {
-	background-color: $blue;
-}
 
-.iconbtndelete:hover {
-	background-color: $red;
+.icon-btn-delete:hover {
+	background-color: $buttonhover !important;
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: for use &:hover in next line like this
.icon-btn-delete {
   &:hover {
   }
}

Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: why are we using important can't we use more selectors important should be the last option

**Review by rahulsavaria — CHANGES_REQUESTED** [review]

File: public-html/asset/scss/components/form.scss (hunk 1 above)
Comment (shriaas2898) [reply public-html/asset/scss/components/form.scss]: Removed !important

Comment (shriaas2898) [reply public-html/asset/scss/components/form.scss]: Done!

File: public-html/asset/scss/components/form.scss [hunk 2]
```diff
@@ -42,16 +42,20 @@ button {
 	padding: 10px 20px;
 }
 
-.main button:hover {
-	background-color: $buttonhover;
+button 
+&:hover {
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: indentation is not correct

File: public-html/asset/scss/components/form.scss [hunk 3]
```diff
@@ -42,16 +42,20 @@ button {
...
+		background-color: $buttonhover;
 }
 
-.main input:hover {
-	background-color: $white;
+.main {
+	input 
+	&:hover {
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: same

File: public-html/asset/scss/components/form.scss [hunk 4]
```diff
@@ -103,21 +83,29 @@ button {
 	width: 50px;
 
 	.fas {
-		@include table-action-icons;
+		color: #fff;
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: use variable

**Review by rahulsavaria — CHANGES_REQUESTED** [review]

**Review by rahulsavaria — APPROVED** [review]

**Pull request merged**